src/Plotting.jl

```diff
@@ -1,13 +1,28 @@
+#  Copyright 2022 California Institute of Technology
```

Comment:
I added the copyright block here since it was missing. Can remove if it shouldn't be here.

Reply:
This works!